BackofficeBundle/Resources/views/form.html.twig

```diff
@@ -11,4 +11,9 @@
     <span class="orchestra-hidden-info" id="dynamic-modal-title">{{ ('open_orchestra_backoffice.block.' ~ blockType ~ '.title') | trans }}</span>
```

Comment:
Not for this line, but for the file above : BackofficeBundle/Resources/public/coffee/table/.tableviewLoader.coffee.swp is useless, it should be removed